validation/oauth_config.xsd

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We'll want a test case that uses this xsd/file test_xml_parser.py and/or test_xsd_validator.py I think.

Reply:
I was looking at an out of date tab, you can possibly ignore this.

Reply:
Added a test case after initial version that tests this, using the existing mcd test as a guide.